src/main/java/com/easypost/exception/API/ServiceUnavailableError.java

```diff
@@ -5,7 +5,7 @@
 import com.easypost.model.Error;
 import com.easypost.exception.EasyPostException;
 
-public class ServiceUnavailablError extends EasyPostException {
+public class ServiceUnavailableError extends EasyPostException {
```

Comment:
Thanks for fixing the typo, this will need to be documented in Changelog

Reply:
Resolved